asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -187,6 +217,8 @@ def load_cache_files(cls, config_kwargs, field):
                     config_cls = SchNetModelConfig
                 case ModelType.e3nn:
                     config_cls = E3NNModelConfig
+                case ModelType.visnet:
+                    config_cls = ViSNetModelConfig
```

Comment:
However in several places we explicitly references `ViSNetModelConfig`. This needs to be taken out and addressed differently in order for `asap-ml` to work with both `mtenn` release version and Github version

Reply:
Just need to make sure these branches do not execute with HAS_VISNET guard.